+++ src/lib/fuse-ext2/e2fsprogs-1.41.12.newgit/ext2fs/test_io.c
@@ -9,6 +9,8 @@
  * %End-Header%
  */
 
+#include "config.h"
+
 #include <stdio.h>
 #include <string.h>
 #if HAVE_UNISTD_H
